turbopack/crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -522,20 +527,23 @@ impl EsmExports {
                     } else {
                         Cow::Borrowed(name.as_str())
                     };
+                    let ctxt = eval_context
+                        .imports
+                        .exports
+                        .get(name)
+                        .map(|id| id.1)
+                        .expect("failed to get the correct SyntaxContext");
```

Comment:
It would seem like `self` (where `name` comes from) and `parsed` (where `eval_context` comes from) must refer to the same code, and therefore `name` should always be in `eval_context`, but turbo-tasks is eventually consistent, so they might not!

Turbo-tasks does not suppress panics (maybe it should when they're a result of eventual consistency...), so we must handle it as a `Result`.

```suggestion
                        .context("failed to get the correct SyntaxContext")?;
```

Reply:
Good to know, thank you!